### DISCUSSION TITLE
Sponsorship and Mentorship Opportunity with CybeRavens FTC Team

### DISCUSSION THREAD

**Discussion by jbearr** [community/community#135287] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

General

### Body

Dear Github,

I hope this message finds you well. My name is Noah Song, and I am writing on behalf of the CybeRavens, FTC Team #26419. We are a dedicated group of high school students committed to excelling in robotics and making a positive impact through STEM education located in the Bay Area.

Our team has been working tirelessly to innovate and create competitive robots for the FIRST Tech Challenge (FTC). This season, we aim to push the boundaries of our capabilities by implementing advanced technologies and enhancing our outreach programs. However, to achieve our goals, we seek the support and mentorship of industry leaders who share our vision for innovation and excellence.

We are reaching out to Github because of your renowned legacy in cutting-edge engineering and design. We believe that a partnership with Github  could provide us with invaluable insights and guidance from your engineers, as well as the financial support necessary to compete at the highest level. Your involvement would not only help us achieve our goals but also inspire the next generation of engineers and innovators.

If Github is interested in exploring sponsorship or mentorship opportunities with the CybeRavens, we would be thrilled to provide you with our detailed sponsorship packet. This document outlines our team’s mission, goals, budget, and the various benefits that come with supporting our team. We have a PDF of the sponsorship packet ready to send at your convenience.

We look forward to the possibility of collaborating with Github and are excited about the potential to learn from your expertise. Please let us know if you would like to discuss this opportunity further or if there are any additional details you require.

Thank you for considering our request. We are hopeful for a positive response and eager to share our journey with you.

Warm regards,

Noah Song
Co-Lead
CybeRavens FTC Team #26419
cyberavensftc@gmail.com

**Comment by ebndev** [comment]
Hi @jbearr!  👋 

Thanks for posting in the Community. The `Sponsors` category is a place for our community to discuss and provide feedback on the GitHub Sponsors feature, not for sponsorship opportunities. Unfortunately, we currently do not allow self-promotion or advertising on the Community Discussions. We want to make sure there is space for users to ask questions without overwhelming them with other conversations. 

We appreciate your understanding!
